kedro-datasets/static/jsonschema/kedro-catalog-1.0.0.json

```diff
@@ -0,0 +1,1863 @@
+{
+    "type": "object",
+    "patternProperties": {
+      "^[a-z0-9-_]+$": {
+        "required": [
+          "type"
+        ],
+        "properties": {
+          "type": {
+            "type": "string",
+            "enum": [
+              "partitions.IncrementalDataset",
+              "partitions.PartitionedDataset",
+              "api.APIDataset",
+              "biosequence.BioSequenceDataset",
+              "dask.CSVDataset",
+              "dask.ParquetDataset",
+              "databricks.ManagedTableDataset",
+              "email.EmailMessageDataset",
+              "geopandas.GenericDataset",
+              "holoviews.HoloviewsWriter",
+              "huggingface.HFDataset",
+              "huggingface.HFTransformerPipelineDataset",
+              "ibis.FileDataset",
+              "ibis.TableDataset",
+              "json.JSONDataset",
+              "matlab.MatlabDataset",
+              "matplotlib.MatplotlibWriter",
+              "networkx.NetworkXDataset",
```

Comment:
Not sure why this is `NetworkXDataset`.. it looks like it should be three different ones instead: gml, graphml and json